### PR TITLE
issues: add iotimeout.md

### PR DESCRIPTION
**Pull request by jzelinskie**

### CODE REVIEW

File: issues/iotimeout.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+frontpage: True
+---
+Attemping to interact with a remote repository yields an `i/o timeout`.
+
+```
+$ sudo docker pull ...
+FATA[0021] Error response from daemon: v1 ping attempt failed with error: Get https://quay.io/v1/_ping: dial tcp: i/o timeout.
+```
+
+Please read over and post in [this docker issue](https://github.com/docker/docker/issues/13337) which is tracking the bug.
```
Comment (josephschorr) [inline issues/iotimeout.md]: "This is a known issue currently in Docker. To track the progress of this issue, please see (link here). Any additional feedback is also appreciated on that issue. "

File: issues/iotimeout.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+A reboot has been seen to resolve the issue.
```
Comment (josephschorr) [inline issues/iotimeout.md]: Add a header before this saying: "Workaround"

**Comment by jzelinskie** [comment]
ptal

**Comment by josephschorr** [comment]
LGTM

**Pull request merged**